src/app/static/src/app/components/plots/bubble/SizeLegend.vue

```diff
@@ -0,0 +1,93 @@
+                        :r="circle.radius" :cx="circle.x" :cy="circle.y"></circle>
+                <text v-for="(circle, index) in circles" :key="'text-' + index" text-anchor="middle"
+                      :x="circle.textX" :y="circle.textY">{{circle.text}}</text>
+            </svg>
+        </div>
+    </l-control>
+</template>
+
+<script lang="ts">
+    import Vue from "vue";
+    import {LControl} from "vue2-leaflet";
+    import {getRadius} from "./utils";
+    import {NumericRange} from "../../../types";
+
+    interface Circle {
+        x: number
+        y: number,
+        radius: number,
+        text: string,
+        textX: number,
+        textY: number
+    }
+
+    interface Props {
+        indicatorRange: NumericRange,
+        minRadius: number,
+        maxRadius: number
+    }
+
+    interface Computed {
+        circles: Circle[],
+        width: number,
+        height: number
+    }
+
+    const numeral = require('numeral');
+
+    export default Vue.extend<{}, {}, Computed, Props>({
+        name: "SizeLegend",
+        props: {
+            "indicatorRange": Object,
+            "minRadius": Number,
+            "maxRadius": Number
+        },
+        components: {
+            LControl
+        },
+        computed: {
+            width: function() {
+                return (this.maxRadius * 2) + 2; //leave room for stroke width
+            },
+            height: function() {
+                return (this.maxRadius * 2) + 10; //leave room for the top text
+            },
+            circles: function() {
+                const x = this.width / 2;
+                const circleFromRadius = (r: number, value: number, under: boolean = false) => {
+                    const y = this.height - r;
+
+                    let text = value > 1000 ? numeral(value).format("0a") : (+value.toFixed(2)).toString();
+                    if (under && text != "0"){
+                        text  = "<" + text;
+                    }
+
+                    return {x: x,  y: y, radius: r, text: text, textX: x, textY: y-r}
+                };
+
+                //We treat the minimum circle differently, since the smallest radius is actually likely to cover quite
+                //a wide range of low outliers, so we show the value for the next pixel up and prefix with '<'
+                const nextMinRadius = this.minRadius + 1;
+                const valueScalePoint = (Math.pow(nextMinRadius,2) - Math.pow(this.minRadius,2))/
+                                            (Math.pow(this.maxRadius, 2) - Math.pow(this.minRadius, 2));
+                const nextValue = (valueScalePoint * (this.indicatorRange.max - this.indicatorRange.min))
+                                            + this.indicatorRange.min;
+                const minCircle = circleFromRadius(this.minRadius, nextValue, true);
+
+                const steps = [0.25, 0.5, 0.75, 1];
```

Comment:
Can steps be a data property of the component?

Also do you think it gives a better sense of scale to use [0.125, 0.25, 0.5, 1], so each bubble is twice the size of the previous? Or [0.1, 0.25, 0.5, 1] even - starting at 0.25 means there's a very big jump from the negligible bubble to the first size, but even 0.125 is quite a big jump.

To compare:

(this is [0.1, 0.25, 0.5, 1])
![Screenshot from 2020-01-10 13-13-19](https://user-images.githubusercontent.com/6736820/72155648-7135d900-33ab-11ea-9b2d-67d9c7079bd0.png)

vs

![Screenshot from 2020-01-10 13-14-35](https://user-images.githubusercontent.com/6736820/72155645-6ed37f00-33ab-11ea-83e5-8716949f8f46.png)

Reply:
Yeah you're right, particularly since in pratice people are mostly going to be looking at the smaller bubbles (those at district level rather than country/region level)